Tests/test_file_mic.py

```diff
@@ -1,10 +1,46 @@
-from helper import unittest, PillowTestCase
+from helper import unittest, PillowTestCase, hopper
 
-from PIL import MicImagePlugin
+from PIL import Image, MicImagePlugin
+
+TEST_FILE = "Tests/images/hopper.mic"
 
 
 class TestFileMic(PillowTestCase):
 
+    def test_sanity(self):
+        im = Image.open(TEST_FILE)
+        im.load()
+        self.assertEqual(im.mode, "RGBA")
+        self.assertEqual(im.size, (128, 128))
+        self.assertEqual(im.format, "MIC")
+
+        im2 = hopper("RGBA")
+        self.assert_image_similar(im, im2, 123.5)
```

Comment:
Epsilon is 123.4919! `im.show()` gives:

![im.show()](https://cloud.githubusercontent.com/assets/1324225/23843932/bc875dec-07c7-11e7-93dc-842f5fb6d34b.png)

`im2.show()` looks like:

![im2.show()](https://cloud.githubusercontent.com/assets/1324225/23843963/ece5a5de-07c7-11e7-85e3-f630a8e19b59.png)


hopper.mic opened in Microsoft Image Composer looks like:
![hopper.mic in MIC](https://cloud.githubusercontent.com/assets/1324225/23843947/d743af64-07c7-11e7-8389-475a2b1bbb79.png)